config.json

```diff
@@ -61,6 +61,15 @@
                 "val": "South America",
                 "bounds": [-100, -70, -30, 25]
             }
+        },
+        {
+            "name": "oceania",
+            "specs": {
+                "src": "admin_0_countries",
+                "key": "continent",
+                "val": "Oceania",
+                "bounds": [-10, -50, -180, 100]
```

Comment:
From https://en.wikipedia.org/wiki/Oceania 

![image](https://user-images.githubusercontent.com/6675409/46084588-ffb02800-c171-11e8-81cd-21a00f39968b.png)

Oceania extends all the way to the [Easter Island at 109°22′W](https://en.wikipedia.org/wiki/Easter_Island).

Reply:
Thank you for the remark! Sorry for the delay as I am having many projects at the same time...
I have modified the file.